ros2_test_cases_stats.py

```diff
@@ -112,24 +191,41 @@ def IsoDate(value: str) -> datetime.date:
 def parse_args():
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        '-t', '--token',
-        required=True,
-        help='Github access token, which must at least have admin:org:read:org and repo:public_repo rights')
+        '--repo',
+        help='Repository in the format <owner>/<repo>. \
+                e.g. osrf/ros2_test_cases or gazebosim/gazebo_test_cases. (default: %(default)s)'
+        , default="osrf/ros2_test_cases")
     parser.add_argument(
-        '-s', '--since',
-        type=IsoDate,
-        required=True,
-        help='Only fetch data since the date specified (in ISO YYYY-MM-DD form)')
+        '--label', required=True,
+        help='Label to filter issues by. e.g. jazzy or ionic')
+    parser.add_argument(
+        '--assignments',
+        action='store_true',
+        help='Assignment of open issues')
+    parser.add_argument(
+        '--raw-output',
+        type=argparse.FileType('w'),
+        help='Output file to save raw JSON output from queries for manual JSON manipulation')
```

Comment:
That's fine, maybe slightly less convenient because we'd have to run the command which hits GitHub servers twice. My typical use case is to save the raw output on disk since I often experiment with the `jq` invokation and wouldn't want to fetch the data from the web everytime.